### PR TITLE
fix: #4206

### PR DESCRIPTION
**Pull request by yoution**
@maxceem please reivew #4206

### CODE REVIEW

File: src/projects/detail/components/SpecQuestions.jsx [hunk 1]
```diff
@@ -389,6 +390,12 @@ class SpecQuestions extends React.Component {
         options: q.options,
       })
       break
+    case 'jobs-picker':
+      ChildElem = JobsPickerQuestion 
+      _.assign(elemProps, {
+        options: q.options,
+      })
```
Comment (maxceem) [inline src/projects/detail/components/SpecQuestions.jsx:397]: @yoution we don't use options for this component, so this can be removed.

**Review by maxceem — CHANGES_REQUESTED** [review]
@yoution thanks for fast fix. I've noticed a few issues.

1. Styles has changed for the text fields. We have to use the same styles as before:

    image
    
2. Skills list is not shown, it should be shown. Probably have to pass some other value for `skillsCategories` maybe `undefined` or `null`. Our aim is to show all skills, not only one category.

    image
    
3. If I add 3 items, and after remove the second item, then in dropdowns I would see the data from the 3rd, items instead of the 2nd item. See demo video https://monosnap.com/file/eWPhOjBU3CNRX091n5ehcYuVXJoqZN

    This issue usually happens because of using `key={jobIndex}`. Instead of this when creating default values for new row, add some new field, like `key = _.uuid('job_key_')` which would be unique, and use it for `key` instead of `key={jobIndex}`.
 
 I could continue reviewing after these 2 issues are fixed.

**Comment by yoution** [comment]
@maxceem  
image
1. the api between  dev and my local , maybe not the same api, 
2. 
image
there is no entry for the job page

**Comment by maxceem** [comment]
image

@yoution this is correct in `tass-jobs-2` branch we use a new endpoint for getting skills without categories.

image

Yeah, we hide it. But you can still use a direct link https://connect.topcoder-dev.com/new-project/taas-jobs

**Comment by maxceem** [comment]
@yoution regarding showing skills. It was not your issue, I've fixed loading skills in the existent code. Please, pull the latest changes from `appirio-tech:feature/taas-jobs-2` I've used the fix there.

**Comment by yoution** [comment]
@maxceem please review again

**Comment by yoution** [comment]
sorry,I forget to change the fix message, I just use the previous message, next time I will pay attention to

**Review by maxceem — APPROVED** [review]
Great @yoution. Everything works good. Merging for testing by QA team.

**Pull request merged**